lib/rzo/app/templates/environment.conf.erb

```diff
@@ -0,0 +1,18 @@
+# Each environment can have an environment.conf file. Its settings will only
```

Comment:
The rizzo generate command adds that inline provision shell command only for puppetmaster nodes.  All the other nodes have the line omitted.

Is that what you meant?